src/component/TurnipCalculator.tsx

```diff
@@ -55,6 +70,17 @@ export default function TurnipCalculator(props: any) {
 
   return (
     <div className={'turnip-calc'}>
+      {alertPrice ? (
```

Comment:
Can you add some tests in `TurnipCalculator.spec.tsx` to ensure that these error messages will appear when price and turnip count are 0 respectively? Also add a couple tests to ensure that these error messages will disappear when we input a number greater than 0? You can use `fireEvent` in react-testing-library to accomplish the text input action for the test case, and you can look at the spec files for other ideas on how to accomplish these test cases.

Reply:
Added it on the new commits.